### PR TITLE
CORE-6656 remove dependencies on ledger-obsolete

### PR DESCRIPTION
**Pull request by blsemo**
Remove the last few straggling dependencies on the old ledger API

API changes: https://github.com/corda/corda-api/pull/567

### CODE REVIEW

File: libs/packaging/packaging/test/contract-cpk/src/main/kotlin/net/corda/libs/packaging/test/contract/Contract.kt [hunk 1]
```diff
@@ -1,15 +1,5 @@
...
-class PackagingTestState : ContractState {
-    override val participants: List<AbstractParty>
-        get() = listOf()
-}
-
-class PackagingTestContract : Contract {
-    override fun verify(tx: LedgerTransaction) {}
-}
\ No newline at end of file
+class PackagingTestContract
```
Comment (vlajos) [inline libs/packaging/packaging/test/contract-cpk/src/main/kotlin/net/corda/libs/packaging/test/contract/Contract.kt:5]: This looks quite empty.
Could the whole file be deleted, perhaps?

Comment (blsemo) [reply libs/packaging/packaging/test/contract-cpk/src/main/kotlin/net/corda/libs/packaging/test/contract/Contract.kt:5]: No idea - the file never did anyting, it's there for some packaging tests to pack something.

Comment (blsemo) [reply libs/packaging/packaging/test/contract-cpk/src/main/kotlin/net/corda/libs/packaging/test/contract/Contract.kt:5]: I think the classes are fine like that - they just have to be there so we got something to package.

**Review by ronanbrowne — DISMISSED** [review]
LGTM Build

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Pull request merged**